graphjet-core/src/main/java/com/twitter/graphjet/datastructures/Pair.java

```diff
@@ -1,4 +1,4 @@
-package com.twitter.graphjet.algorithms;
+package com.twitter.graphjet.datastructures;
```

Comment:
Right, in that case I'm not sure why using a pair is better than returning a class that can be more easily interpreted. Might be worth thinking about whether there is solution that avoids the object creation if that shows up as an overhead while profiling...